documentation/user-interface/fixture-editor.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+title: Fixture Editor
+permalink: /documentation/user-interface/fixture-editor.html
+---
+# Fixture Editor
+
+The **Fixture** editor allows you to view and modify a fixture. The editor opens when you double-click the fixture in the [Explorer](explorer-panel.md) panel.
+
+The editor consists of the following parts:
+
+* The toolbar -Provides information about the fixture (the name and tested webpage's URL) and buttons for performing [typical tasks](#typical-tasks) with the fixture.
+
+    The webpage's URL is clickable and leads to the webpage opened in the browser.
+
+* The test list - Displays tests included to the fixture. The tests' rows are clickable and lead to the [Test](test-editor.md) editor.
+
+![Fixture editor](../../images/user-interface/fixture-editor.png)
+
+## Typical Tasks
+
+You can perform the following tasks in the editor:
+
+* [Record New Tests](#record-new-tests)
+* [Run the Fixture or Individual Tests](#run-the-fixture-or-individual-tests)
+* [Convert the Fixture to JavaScript](#convert-the-fixture-to-JavaScript)
+* [Modify the Fixture's Properties](#modfy-the-fixtures-properties)
+* [Delete Tests](#delete-tests)
+
+### Record New Tests
+
+Click the ![Record button](../../images/working-with-testcafe-studio/record-test-icon.png) **Record a new test** button on the editor's toolbar to start recording a new test and add it to the fixture.
+
+### Run the Fixture or Single Test
+
+Click the ![Run tests button](../../images/working-with-testcafe-studio/action-run-icon.png) **Run all tests** button on the editor's toolbar to run the whole fixture.
```

Comment:
I can't remember Armina's words about `whole vs entire` quite well but I suspect here you should say `entire`